### DISCUSSION TITLE
Add workflow scope

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#43236] [Mobile] [answered]
It is currently impossible to edit a workflow file because the GitHub mobile app does not contain the workflow scope. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone 12 mini (iOS 16.2)
</details>

**Comment by HenSquared** [comment]
Hi, 

We'd like to support editing workflow files, but haven't completed the work to do that yet. I've added this feedback to an internal tracking issue discussing the work involved. Thanks for the feedback! This helps us prioritize what our users want.

**Comment by spaltrowitz** [comment, marked as answer]
Hi @hfhbd, thanks for your feedback. A feature related to your feedback has now shipped! 🚢 https://github.blog/changelog/2023-05-11-edit-workflow-files-on-github-mobile/ cc @HenSquared
